packages/twenty-server/src/engine/metadata-modules/flat-object-metadata/services/flat-object-metadata-validator.service.ts

```diff
@@ -32,6 +33,40 @@ export class FlatObjectMetadataValidatorService {
     private readonly flatFieldMetadataValidatorService: FlatFieldMetadataValidatorService,
   ) {}
 
+  public validateFlatObjectMetadataUpdate({
```

Comment:
looking at the function name, I would expect this method to validate the whole update. It seems to handle only specificicty of update?

in this case I would add "Specific" or equivalent in the naming